### DISCUSSION TITLE
Show Request/Response bodies in timeline

### DISCUSSION THREAD

**Discussion by gpasq** [Kong/insomnia#3891] [Ideas]
Without request and response bodies in the timeline, it's very difficult to show a transcript of the session to the service developer.  If those were present, and simply copy and paste would make the  entire call transcript available to be used by the API developer to see exactly what was sent and received in the context of the overall call.

**Comment by dimitropoulos** [comment]
it will speed the development of this along if you could provide a sample request (e.g. to mockbin.org) and show a before/after of what you'd like the timeline to show.  that way we can be totally sure we're implementing the thing you're actually asking for, you know what I mean?
